analyzers/src/SonarAnalyzer.ShimLayer/SyntaxKindEx.cs

```diff
@@ -79,21 +82,22 @@ public static class SyntaxKindEx
         public const SyntaxKind NullableDirectiveTrivia = (SyntaxKind)9055;
         public const SyntaxKind FunctionPointerType = (SyntaxKind)9056;
         public const SyntaxKind FunctionPointerParameter = (SyntaxKind)9057;
-        public const SyntaxKind FunctionPointerParameterList = (SyntaxKind)9058;
-        public const SyntaxKind FunctionPointerCallingConvention = (SyntaxKind)9059;
+        public const SyntaxKind FunctionPointerParameterList = (SyntaxKind)9058; // Sonar
+        public const SyntaxKind FunctionPointerCallingConvention = (SyntaxKind)9059; // Sonar
         public const SyntaxKind InitAccessorDeclaration = (SyntaxKind)9060;
         public const SyntaxKind WithExpression = (SyntaxKind)9061;
         public const SyntaxKind WithInitializerExpression = (SyntaxKind)9062;
-        public const SyntaxKind RecordClassDeclaration = (SyntaxKind)9063;
+        public const SyntaxKind RecordDeclaration = (SyntaxKind)9063; // Sonar
```

Comment:
Changed to sync with StyleCop (and the SyntaxKind name based on [Microsoft documentation](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntaxkind?view=roslyn-dotnet-4.3.0))